content/video-production/live-events.mdx

```diff
@@ -1,14 +1,14 @@
 ---
 seo:
-  title: Live Events
+  title: 'Filming, Livestreaming, Recording & Video Editing'
   description: >-
     SSW Consulting has over 30 years of Microsoft software and web development
```

Comment:
1. Update the SEO description to:
Our live streaming services feature top-notch multi-camera setups to ensure a flawless audience experience.

Reply:
Done